packages/components/package.json

```diff
@@ -49,8 +49,8 @@
     "shortid": "^2.2.16"
   },
   "peerDependencies": {
-    "react": "^17.x",
-    "react-dom": "^17.x"
+    "react": "^16.8.0 || ^17.0.0-rc.1 || ^18.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0-rc.1 || ^18.0.0"
```

Comment:
Just use `>= 16.8.0`? No need for the 17 release candidates IMO

I use https://semver.npmjs.com/ to check what the actual published versions are that will match my version range

Reply:
Seems to be what other libs do: https://github.com/atlassian/react-beautiful-dnd/blob/013bfceac04ff48548c33cdc468dd2927446fc1b/package.json#L148
Though I think you're right, doesn't really seem necessary at this point...

Reply:
I'm going to leave this in another change with #1493 and #1863 , I'll just update the docs with this PR.